src/main/kotlin/com/terraformation/backend/admin/AdminDocumentController.kt

```diff
@@ -22,7 +22,7 @@ import org.springframework.web.multipart.MultipartFile
 import org.springframework.web.servlet.mvc.support.RedirectAttributes
 
 @Controller
-@RequestMapping("/admin")
+@RequestMapping("/admin/document-producer")
```

Comment:
Good catch.

This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.